src/main/java/com/cgdp/library/controller/AuthorController.java

```diff
@@ -0,0 +1,30 @@
+package com.cgdp.library.controller;
+
+import com.cgdp.library.dto.author.AuthorDTO;
+import com.cgdp.library.dto.author.CreateAuthorResponseDTO;
+import com.cgdp.library.dto.author.CreateAuthorRequestDTO;
+import com.cgdp.library.service.AuthorService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import javax.validation.Valid;
```

Comment:
replace javax by jakarta
```suggestion
import jakarta.validation.Valid;
```